pkg/cmd/update-tls-artifacts/metadata/owner.go

```diff
@@ -0,0 +1,146 @@
+package metadata
+
+import (
+	"fmt"
+
+	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+func newOwnerViolation(name string, pkiInfo *certgraphapi.PKIRegistryInfo) (Violation, error) {
+	registry := &certgraphapi.PKIRegistryInfo{}
+
+	for i := range pkiInfo.CertKeyPairs {
+		curr := pkiInfo.CertKeyPairs[i]
+		owner := curr.CertKeyInfo.OwningJiraComponent
+		if len(owner) == 0 || owner == unknownOwner {
+			registry.CertKeyPairs = append(registry.CertKeyPairs, curr)
+		}
+	}
+	for i := range pkiInfo.CertificateAuthorityBundles {
+		curr := pkiInfo.CertificateAuthorityBundles[i]
+		owner := curr.CABundleInfo.OwningJiraComponent
+		if len(owner) == 0 || owner == unknownOwner {
+			registry.CertificateAuthorityBundles = append(registry.CertificateAuthorityBundles, curr)
+		}
+	}
+
+	v := Violation{
+		Name:     name,
+		Registry: registry,
+	}
+
+	markdown, err := generateMarkdownNoOwner(pkiInfo)
+	if err != nil {
+		return v, err
+	}
+	v.Markdown = markdown
+
+	return v, nil
+}
+
+func generateMarkdownNoOwner(pkiInfo *certgraphapi.PKIRegistryInfo) ([]byte, error) {
```

Comment:
Is the content of this file built the same way kind of violation?

violations.md

```
This is a list of violations for <insert type here>

Table of Contents

<OwningComponent> (<number of violations>)
1. ordered list of violations with descriptions associated with each item

```